apps/web/components/apps/AppList.tsx

```diff
@@ -119,7 +119,10 @@ export const AppList = ({ data, handleDisconnect, variant, listClassName }: AppL
             </div>
           ) : null
         }>
-        <AppSettings slug={item.slug} />
+        <AppSettings
+          slug={item.slug}
+          credentialId={item.credentialOwner?.credentialId || item.userCredentialIds[0]}
+        />
```

Comment:
I need credentialId available in AppSettings so that credential information can be seen and updated in the credentials form. Only way to get this information is by drilling it down from this component.